src/pages/DatasetRegistration.js

```diff
@@ -16,34 +16,25 @@ import * as fp from 'lodash/fp';
 import './DataAccessRequestApplication.css';
 
 
-const noOptionMessage = 'Start typing a Dataset Name, Sample Collection ID, or PI';
+// const noOptionMessage = 'Start typing a Dataset Name, Sample Collection ID, or PI';
 
 class DatasetRegistration extends Component {
 
   constructor(props) {
     super(props);
-    this.dialogHandlerSave = this.dialogHandlerSave.bind(this);
-    this.setShowDialogSave = this.setShowDialogSave.bind(this);
     this.state = {
       nihValid: false,
       disableOkBtn: false,
       showValidationMessages: false,
-      optionMessage: noOptionMessage,
       file: {
         name: ''
       },
+      properties: {},
       showModal: false,
-      completed: '',
       showDialogSubmit: false,
-      showDialogSave: false,
-      step: 1,
       formData: {
```

Comment:
I'm wondering if I should further split formData into 2 groups for datasetData and consentData to keep it organized?

Reply:
Agreed, that makes a lot of sense